src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java

```diff
@@ -119,12 +121,18 @@ public String getDisplayName() {
     @Override
     public String getIconFileName() {
         // This _needs_ to be done in getIconFileName only because of JENKINS-33683.
-        Jenkins jenkins = Jenkins.getInstance();
-        if (!jenkins.hasPermission(Jenkins.ADMINISTER)) return null;
+        Jenkins jenkins = Jenkins.get();
+        if (!jenkins.hasPermission(SystemReadPermission.SYSTEM_READ)) return null;
```

Comment:
Since we anticipate to change these in bulk, please use `CloudStatistics#getRequiredPermission()` here and in the groovy views to avoid the duplicity.

Reply:
Addressed this in https://github.com/jenkinsci/cloud-stats-plugin/pull/22/commits/3e7ba21073b5f0590966ee0d90246ad558c0fa7a